### ISSUE TITLE
General : Unable to join the Grafana slack community channel

### ISSUE THREAD

**Issue by kapil111999** [grafana/loki#7452]
I am getting redirected to a site called raintank.com, where I it is asking me to sign up. Please fix this. 
![image](https://user-images.githubusercontent.com/52229887/196445731-42a43882-03a8-40d3-8a23-1b58593caefb.png)

**Comment by gelicia** [comment]
@kapil111999 what link were you following to get to this page?

**Comment by JStickler** [comment]
https://slack.grafana.com/   Is the link you want.  
FYI - Raintank = Grafana Labs.

**Comment by smbecker** [comment]
> https://slack.grafana.com/ Is the link you want. FYI - Raintank = Grafana Labs.

I am unable to join the slack channel using this link

![image](https://github.com/grafana/loki/assets/125765/dd5cb051-0a34-4c82-a8ec-fac6b2da5ab1)

**Comment by JStickler** [comment]
My mistake, it's [grafana.slack.com](grafana.slack.com) for the community slack.  The previous URL I shared is our internal Slack. Not confusing at all having the two URLs be so similar.  🙄

**Comment by smbecker** [comment]
The `README.md` should be updated as well https://github.com/grafana/loki/blob/main/README.md?plain=1#L5

**Comment by smbecker** [comment]
Also, how do you request access to the slack channel? All it gives me is the ability to log in with an existing account.

**Comment by JStickler** [comment]
I'll update the README.  Thanks for pointing out that we had it wrong there!

>Also, how do you request access to the slack channel? 
I've had a Slack account for so long, I'm not sure I remember how the initial setup works.  I'd try the Slack help.  Once you're in the Grafana workspace, you should be able to search and join channels.
This might help? https://slack.com/help/articles/212675257-Join-a-Slack-workspace

**Comment by smbecker** [comment]
Yeah I have a slack account but I am not able to join the grafana workspace. According to [this](https://community.grafana.com/faq#civilized), you need to use [slack.grafana.com](https://slack.grafana.com) to request access to the workspace but as mentioned above, that site doesn't appear to be functional right now.

**Comment by smbecker** [comment]
And with that in mind, then the readme shouldn't be updated since that is the proper location for requesting access. It's just not functional right now.

**Comment by smbecker** [comment]
```sh
curl http://slack.grafana.com
```
responds with
```html
<!DOCTYPE html>
<html>
  <head>
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <meta charset="utf-8">
    <title>Heroku | Application Error</title>
    <style media="screen">
      html,body,iframe {
        margin: 0;
        padding: 0;
      }
      html,body {
        height: 100%;
        overflow: hidden;
      }
      iframe {
        width: 100%;
        height: 100%;
        border: 0;
      }
    </style>
  </head>
  <body>
    <iframe src="//www.herokucdn.com/error-pages/no-such-app.html"></iframe>
  </body>
</html>
```
so, looks like something is messed up with the heroku app/connection

**Comment by alvinlin123** [comment]
Hopefully this get fixed soon!

**Comment by alvinlin123** [comment]
Alright, looks like https://slack.grafana.com/ is fixed now :)

**Comment by JStickler** [comment]
Closing as stale. This issue was opened against a version of Loki that is no longer supported. If this issues still exists in more recent versions of Loki, please leave a comment and we can reopen the issue.
